charts/quay-obj-backup/templates/prometheus-rule.yaml

```
@@ -0,0 +1,19 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "quay-obj-backup.fullname" . }}-rules
+  namespace: {{ .Values.backupNamespace }}
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+spec:
+  groups:
+  - name: cronjob.rules
+    rules:
+    - alert: QuayBackupJobFailure
+      expr: kube_job_status_failed{namespace="{{ .Values.backupNamespace }}", job_name=~"{{ include "quay-obj-backup.fullname" . }}-.*"} > 0
+      labels:
+        severity: error
```

Ospravedlnujem sa, ze som nebol v predchadzajucom komentari presenejsi. Podla platnych pravidiel
https://aspecta.atlassian.net/wiki/spaces/SP/pages/348094498/Labeling+Rules mame zatial len 2 severity - critical a warning. Poprosim preto o zmenu na warning

```suggestion
        severity: warning
```

---

updated